gallery/src/components/NxToast/NxToastFullPageSimpleExample.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { useState } from 'react';
+import {reject, propEq} from 'ramda';
+import {
+  NxGlobalSidebarNavigation,
+  NxGlobalSidebarNavigationLink,
+  NxButton,
+  NxBackButton,
+  NxButtonBar,
+  NxStatefulGlobalSidebar,
+  NxPageMain,
+  NxPageTitle,
+  NxH1,
+  NxToastContainer,
+  NxToast,
+  NxSuccessAlert,
+  NxWarningAlert,
+  NxErrorAlert,
+  NxInfoAlert,
+  NxTile,
+  NxP
+} from '@sonatype/react-shared-components';
+
+import {
+  faArrowLeft,
+  faArrowRight,
+  faLink
+} from '@fortawesome/free-solid-svg-icons';
+
+const sidebarLogoPath = require('../../assets/images/logo-plaid-villain-text.png');
+
+type ToastTypes = 'success' | 'error' | 'info' | 'warning';
+
+interface ToastModel {
+  id: number;
+  type: ToastTypes;
+  message: string;
+}
+
+export default function NxToastFullPageSimpleExample() {
+  const [toastIdInc, setToastIdInc] = useState<number>(0);
+  const [toasts, setToasts] = useState<ToastModel[]>([]);
+
+  const addToast = (type: ToastTypes, message: string) => {
```

Comment:
One way to simplify the example a little bit would be to have `addToast` take the `Nx*Alert` react component constructor itself as the first argument instead of a `ToastType`.

Reply:
> One way to simplify the example a little bit would be to have `addToast` take the `Nx*Alert` react component constructor itself as the first argument instead of a `ToastType`.

Would you be able to clarify what you mean by the NxAlert react component constructor?

Reply:
> Would you be able to clarify what you mean by the NxAlert react component constructor?

So instead of `addToast('success', ...)` it would be `addToast(NxSuccessAlert, ...)`

Reply:
> So instead of `addToast('success', ...)` it would be `addToast(NxSuccessAlert, ...)`

Thanks for the clarification. Cleaned it up a bit here : [0fd7746](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/0fd7746a0d87a82598bd72aa67a7282636fd4de6) and here : [f2fc205](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/f2fc2056f960a83a2aa558d136ce3700fbc452c6). If that looks ok then I will change the other example as well.